README.md

```diff
@@ -20,7 +20,7 @@ run distributed or non-distributed TensorFlow jobs on Kubernetes.
 ## Installation
 
 ```bash
-kustomize build manifests/overlays/standalone | kubectl apply -f -
+kustomize build "github.com/kubeflow/tf-operator.git/manifests/overlays/standalone?ref=v1.1.0" | kubectl apply -f -
```

Comment:
Should this be pointing master branch instead?

Reply:
Master branch could contain unstable changes. It would be safer to point to a stable version, but it does take a bit more work during the release process to have to update this.

Reply:
@terrytangyuan WDYT

Reply:
Perhaps we can include instructions for installing both dev version and stable version?

Reply:
Sounds good, updated